performance-timeline/po-navigation.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>PerformanceObservers: navigation</title>
```

Comment:
Reading through, lgtm. We don't support this in Chrome yet though because we're still on NT1.